dev/tests/verification/Resources/ExtendParentDataTest.txt

```diff
@@ -27,9 +27,9 @@ class ExtendParentDataTestCest
 	public function ExtendParentDataTest(AcceptanceTester $I)
 	{
 		$I->createEntity("simpleDataKey", "test", "extendParentData", [], []); // stepKey: simpleDataKey
-		$I->searchAndMultiSelectOption("#selector", ["otherName"]); // stepKey: getName
+		$I->searchAndMultiSelectOption("#selector", ["name", "otherName"]); // stepKey: getName
```

Comment:
This is not an array, should this result in a merge and replace instead? I wonder if this could break existing tests

Reply:
Yes, possibly it can cause a test failure, but I'll fix them if any